Evtx/BinaryParser.py

```diff
@@ -186,7 +186,10 @@ def dosdate(dosdate, dostime):
 
 def parse_filetime(qword):
     # see http://integriography.wordpress.com/2010/01/16/using-phython-to-parse-and-present-windows-64-bit-timestamps/
-    return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    try:
+        return datetime.utcfromtimestamp(float(qword) * 1e-7 - 11644473600)
+    except ValueError:
+        return
```

Comment:
i don't like this. since python is a dynamic language, i've learned (though lots of trials and errors) that its important to keep a single type of data in each variable/return value, or bugs become very hard to track down.

functions *should* return only a single datatype, and *should not* use `None` to signal an exceptional/error condition. otherwise, we'll have to include `if ret is None:` after every function call. thats messy and ugly.

i'd prefer to raise an exception when the timestamp is invalid. so actually keeping it unwrapped is probably ok.

make sense? i welcome discussion and an open to persuasion, so let me know if you disagree strongly.